.github/workflows/container_image.yaml

```diff
@@ -18,12 +18,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@v3
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v3
 
     - run: |
         docker login -u="${{ secrets.QUAY_USERNAME }}" -p="${{ secrets.QUAY_TOKEN }}" quay.io
-        docker buildx build --platform linux/amd64,linux/arm64 -t ${{ env.image_tag_branch }} --push .
-        docker buildx build --platform linux/amd64,linux/arm64 -t ${{ env.image_tag_commit }} --label quay.expires-after=4w --push .
+        make docker-buildx IMG=${{ env.image_tag_branch }}
+        make docker-buildx IMG=${{ env.image_tag_branch }} DOCKER_BUILD_ARGS="--label quay.expires-after=4w"
```

Comment:
See below where I've noted that the `docker-buildx` target both creates and destroys a single-use builder. This means we're going to have no caching between these 2 builds. This is bad for at least 2 reasons:
* It takes twice as long to build
* Because we don't have reproducible builds, they're going to be subtly different

My instinct is to remove builder management from the `docker-buildx` target and just have it use the default builder. If we needed to (which we don't) we could add a wrapper.

Reply:
Ack, I've now dropped the builder.